feature-libs/cart/main/root/config/cart-config.ts

```diff
@@ -15,6 +15,9 @@ export abstract class CartConfig {
     validation?: {
       enabled?: boolean;
     };
+    storageType?:
```

Comment:
We have ngrx state, so it seems there is no scenario that we need sync data in `sessionStorage`, right? I guess that's why we don't have `storageType` in any config.

Reply:
I'll remove it.  We'll let the multi cart persistent service impose the storage type since only the local storage makes sense for storing thee actiive cart.